packages/components/Button/src/Button.android.tsx

```diff
@@ -66,6 +66,7 @@ export const Button = compose<IButtonType>({
     root: View,
     ripple: Pressable,
     stack: { slotType: View, filter: filterViewProps },
+    // eslint-disable-next-line @typescript-eslint/ban-types
     icon: { slotType: Icon as React.ComponentType<object> },
```

Comment:
Are these comments things that are probably worth a followup from our side?

Reply:
So this is the error:
```
ERR! /~/packages/utils/test-tools/src/enzymeTests.ts
ERR!   12:10  error    Don't use `object` as a type. The `object` type is currently hard to use ([see this issue](https://github.com/microsoft/TypeScript/issues/21732)).
ERR! Consider using `Record<string, unknown>` instead, as it allows you to more easily inspect and use the keys  @typescript-eslint/ban-types
```
It's probably worth taking a look. I know too little of the code base to have an opinion.

Reply:
Oh, interesting. That sounds very typescript specific, and is good to know. Thanks!